pkg/util/orchestrator/pod.go

```diff
@@ -127,6 +129,8 @@ func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterID
 		})
 	}
 
+	cluster.SetCacheStats(len(podList), len(podMsgs), orchestrator.K8sPod)
```

Comment:
Thinking about this, it is probably better if you move this common func into this `orchestrator` package as it is shared between `corechecks/cluster` and `orchestrator/pod`, one only for the DCA while the latter for the agent and DCA

Reply:
Thanks for the feedback, I move this function to `pkg/orchestrator`